semgrep/tests/conftest.py

```diff
@@ -167,10 +141,11 @@ def _run_semgrep(
     strict: bool = True,
     quiet: bool = False,
     env: Optional[Dict[str, str]] = None,
-    fail_on_nonzero: bool = True,
+    assert_exit_code: Union[None, int, Set[int]] = 0,
     settings_file: Optional[str] = None,
     force_color: Optional[bool] = None,
     assume_targets_dir: bool = True,  # See e2e/test_dependency_aware_rule.py for why this is here
+    force_metrics_off: bool = True,
```

Comment:
Do we have a test that verifies metrics works now that everything might have it as off?

Reply:
`unit/test_metrics` and `e2e/test_metrics` are pretty extensive and I had to set `force_metrics_off=False` to get those tests to pass

Reply:
Nvm i see the test_metrics sets some to on